airflow/www/views.py

```diff
@@ -2263,6 +2269,45 @@ def ordering(dag):
             # We want to sort by currently running -> active DAGs -> next run time -> alphabetically
             return is_not_running, dag.is_paused, next_run_date, dag.dag_id
 
+        def get_coatue_today(dttm):
+            local_dttm = CLIENT_TIMEZONE.convert(dttm)
+            # If we are not yet past the ROLLOVER_HOUR today, return date time corresponding to today's ROLLOVER_HOUR
+            if local_dttm.hour < DAY_ROLLOVER_HOUR:
+                return local_dttm.replace(hour=DAY_ROLLOVER_HOUR, minute=0, second=0, microsecond=0)
+            # else return datetime corresponding to the ROLLOVER_HOUR tomorrow
+            else:
+                return local_dttm.add(days=1).replace(hour=DAY_ROLLOVER_HOUR, minute=0, second=0, microsecond=0)
+
+        def will_dag_run_in_interval(dag_obj, get_interval_end):
+            now = pendulum.now('UTC')
+            next_run_date = dag_obj.get_next_run_date(now)
+            return (not dag_obj.is_paused) and next_run_date < get_interval_end(now) if next_run_date else False
+
+        job_filter_clauses = {
+            "currently_running": lambda dag_obj: dag_obj.get_num_active_runs() > 0,
+            "disabled": lambda dag_obj: dag_obj.is_paused,
+            "high_priority": lambda dag_obj: dag_obj.pagerduty_priority_level == 'HIGH',
+            "today": lambda dag_obj: will_dag_run_in_interval(dag_obj, get_coatue_today),
+            "upcoming": lambda dag_obj: will_dag_run_in_interval(dag_obj, lambda dttm: dttm.add(hours=1)),
+        }
+
+        job_filter_descriptions = {
+            "currently_running": "Currently Running",
+            "disabled": "Disabled",
+            "high_priority": "High Priority",
+            "today": f"Between now and {DAY_ROLLOVER_HOUR}:00 EST",
```

Comment:
Just make this simple string:
```suggestion
            "today": "Next 24 hrs",
```